util/src/main/i18n/translations/ui_ca_ES.properties

```diff
@@ -37,9 +37,9 @@ setting.visibility = Visibilitat
 # {0} = status (e.g. "Shown" or "Hidden")
 setting.visibility.lore = Visibilitat dels observadors\:{0}
 
-setting.visibility.shown = Mostrar
+setting.observers.value.on = Mostrar
```

Comment:
I don't think we typically fix non-English language translations until we merge Crowdin, but I think this should be ok. Leave it in. (Crowdin has "memory" and does this for us later)